tailwind.config.js

```diff
@@ -197,6 +197,7 @@ module.exports = {
             "100": "100vh",
         },
         screens: {
+            xs: { max: "639px" },
```

Comment:
Whoa! That makes sense, though, that adding a new breakpoint results in all associated selectors to be pulled in as well. I'll back this out.

> I do see what the margin/padding adjustment using this variant is doing, though -- I'm just not sure why this difference is especially necessary.

I am going to revisit this. I did it the tailwind-way to avoid declaring an inline media query. But I definitely don't want us to add a whole slew of unnecessary selectors. I wanted to use this breakpoint, as you rightly identified, to size the nav menu on mobile appropriately. Otherwise, the nav menu on mobile won't be full-width. Note that I am using the same `ul` for both desktop and mobile. They aren't separate like how our current desktop/mobile nav is in `pulumi-hugo`.

Reply:
So I had a thought about this, wouldn't https://github.com/pulumi/docs/pull/6490 remove the unused TW selectors?

Reply:
Despite my previous comment, I am still going to revisit this, by the way.

Reply:
Yes -- for the docs site. But by defining a breakpoint like this here in the theme, we're also implying it'd be okay to use `xs` in a general way, and I don't think we have a case for that. 

Separate from that, though, it's worth pointing out that these breakpoint utilities are set up in a "mobile first" way to build _up_ from a min-width of zero, which includes all widths below `sm`, or `min-width: 640px`. So technically, even if we did want to do something different below 640px, we could just apply that treatment as the base case, and then at 640px and above, behave differently:

```
.logo-nav-menu {
    @apply ... pt-2 -ml-2;

        @screen sm {
            @apply pt-0 -ml-0;
        }

        @screen md {
            @apply pb-2;
        }
    
        ...
``` 

My hunch, though, is that we probably don't need to do anything special below 640 -- but if we did, ☝️ this would be the way to do it "with the grain", so to speak.

Reply:
I ended up doing away with what I originally had and went with exactly what you suggested as well. Mobile by default and adjust for large screens with the screen queries.